### ISSUE TITLE
[open set domain adaptation] Use test image when training

### ISSUE THREAD

**Issue by lianqing11** [VisionLearningGroup/visda-2018-public#12]
Can we use test image when training the domain adaptation model?

**Comment by xcpeng** [comment]
Yes, you can use the test images

Regards,
--
Xingchao Peng, Ph.D. Student
Image and Video Computing Group, Boston University

**Comment by lianqing11** [comment]
Thanks for your respond.

I am curious why choose this kind of setting that at the final stage we can use all of the target image to test our model instead of splitting the target image to train and test set and only can use the train set when training. Because at a real scenario, we can't get the test image when we train the model.

**Comment by xcpeng** [comment]
Hi, Lianqing

That's really a good idea. I would say as most of the Unsupervised Domain
Adaptation in the research community use this setting, i.e. training data
with labels and testing data without labels, we follow this schema.

But we will take consideration of your suggestions in the future. Thanks

Regards,
--
Xingchao Peng, Ph.D. Student
Image and Video Computing Group, Boston University
